analyses/TX_cd_2010/01_prep_TX_cd_2010.R

```diff
@@ -0,0 +1,118 @@
+###############################################################################
+# Download and prepare data for `TX_cd_2010` analysis
+# © ALARM Project, December 2022
+###############################################################################
+
+suppressMessages({
+    library(dplyr)
+    library(readr)
+    library(sf)
+    library(redist)
+    library(geomander)
+    library(cli)
+    library(here)
+    library(sf)
+    library(tidyverse)
+    devtools::load_all() # load utilities
+})
+
+# Download necessary files for analysis -----
+cli_process_start("Downloading files for {.pkg TX_cd_2010}")
+
+path_data <- download_redistricting_file("TX", "data-raw/TX", year = 2010, overwrite = TRUE)
```

Comment:
Can we remove the `overwrite = TRUE`? That should not be necessary for replication, as it will download the data without having to replace anything.